recipes/maturin/meta.yaml

```
@@ -0,0 +1,43 @@
+{% set name = "maturin" %}
+{% set version = "0.7.7" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 2533c648aa8fa19f7a9d4f62282b4adce2f9c07c49f512005f92911be767555e
+
+build:
+  number: 0
+  skip: True  # [py27]
+  missing_dso_whitelist:          # [osx]
+    - /usr/lib/libresolv.9.dylib  # [osx]
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv"
+
+requirements:
+  build:
+    - {{ compiler('c') }}              # [unix]
+    - {{ compiler('m2w64_c') }}        # [win]
+    - rust
+  host:
+    - pip
+    - python
+  run:
+    - python
+
+test:
+  commands:
+    - maturin --help
+
+about:
+  home: https://github.com/PyO3/maturin
+  license: MIT
+  license_family: MIT
+  license_file: license-mit
+  summary: "Build and publish crates with pyo3, rust-cpython and cffi bindings as well as rust binaries as python packages"
```

Would suggest wrapping this text. Possibly like so?

```suggestion
  summary: >-
    Build and publish crates with pyo3, rust-cpython and cffi bindings
    as well as rust binaries as python packages
```

---

Thank you!